benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/timing.cpp

```diff
@@ -0,0 +1,347 @@
+#include <sys/time.h>
```

Comment:
It would be nice if we could reduce the code duplication by somehow replacing `psykal_c_opencl/timing.c` and this file with a single implementation. However, that's (even more) work and I think we'll just live with the duplication for the moment.

Reply:
I prefer solving this in a separate PR as I want to discuss first what is the best solution. I created issue #58 for it.